packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -298,3 +298,11 @@ export const generateComponentHTML = (tag: string, props: TagProps = {}) => {
   }
   return e;
 };
+
+export const generateLongTextAnswer = () => {
+  let longText = '';
+  for (let i = 0; i < 200; i++) {
+    longText += 'This is my long answer. ';
+  }
+  return longText.trim();
+};
```

Comment:
Have fun and try `new Array(200).fill('This is a long answer').join(' ')`

![image](https://github.com/coveo/ui-kit/assets/45688129/1a129dbd-e96a-4d41-9afd-a3aded9cdcb0)

Reply:
So unoriginal.... 

```
const testLongText = `Call me Ishmael. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would sail about a little and see the watery part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to sea as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the ship. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the ocean with me.
There now is your insular city of the Manhattoes, belted round by wharves as Indian isles by coral reefs—commerce surrounds it with her surf. Right and left, the streets take you waterward. Its extreme downtown is the battery, where that noble mole is washed by waves, and cooled by breezes, which a few hours previous were out of sight of land. Look at the crowds of water-gazers there.`;
```